wled00/src/font/clear_3x5.h

```diff
@@ -0,0 +1,141 @@
+    0xE0, 0x80, 0xE0, 0x80, 0xE0,       /* 0x45 E */
+    0xE0, 0x80, 0xE0, 0x80, 0x80,       /* 0x46 F */
+    0x60, 0x80, 0xE0, 0xA0, 0x60,       /* 0x47 G */
+    0xA0, 0xA0, 0xE0, 0xA0, 0xA0,       /* 0x48 H */
+    0xE0, 0x40, 0x40, 0x40, 0xE0,       /* 0x49 I */
+    0x20, 0x20, 0x20, 0xA0, 0x40,       /* 0x4A J */
+    0xA0, 0xA0, 0xC0, 0xA0, 0xA0,       /* 0x4B K */
+    0x80, 0x80, 0x80, 0x80, 0xE0,       /* 0x4C L */
+    0xA0, 0xE0, 0xE0, 0xA0, 0xA0,       /* 0x4D M */
+    0xA0, 0xE0, 0xE0, 0xE0, 0xA0,       /* 0x4E N */
+    0x40, 0xA0, 0xA0, 0xA0, 0x40,       /* 0x4F O */
+    0xC0, 0xA0, 0xC0, 0x80, 0x80,       /* 0x50 P */
+    0x40, 0xA0, 0xA0, 0xE0, 0x60,       /* 0x51 Q */
+    0xC0, 0xA0, 0xE0, 0xC0, 0xA0,       /* 0x52 R */
+    0x60, 0x80, 0x40, 0x20, 0xC0,       /* 0x53 S */
+    0xE0, 0x40, 0x40, 0x40, 0x40,       /* 0x54 T */
+    0xA0, 0xA0, 0xA0, 0xA0, 0x60,       /* 0x55 U */
+    0xA0, 0xA0, 0xA0, 0x40, 0x40,       /* 0x56 V */
+    0xA0, 0xA0, 0xE0, 0xE0, 0xA0,       /* 0x57 W */
+    0xA0, 0xA0, 0x40, 0xA0, 0xA0,       /* 0x58 X */
+    0xA0, 0xA0, 0x40, 0x40, 0x40,       /* 0x59 Y */
+    0xE0, 0x20, 0x40, 0x80, 0xE0,       /* 0x5A Z */
+    0xE0, 0x80, 0x80, 0x80, 0xE0,       /* 0x5B bracketleft */
+    0x80, 0x40, 0x20, 0x00, 0x00,       /* 0x5C backslash */
+    0xE0, 0x20, 0x20, 0x20, 0xE0,       /* 0x5D bracketright */
+    0x40, 0xA0, 0x00, 0x00, 0x00,       /* 0x5E asciicircum */
+    0xE0, 0x00, 0x00, 0x00, 0x00,       /* 0x5F underscore */
+    0x80, 0x40, 0x00, 0x00, 0x00,       /* 0x60 grave */
+    0xC0, 0x60, 0xA0, 0xE0, 0x00,       /* 0x61 a */
+    0x80, 0xC0, 0xA0, 0xA0, 0xC0,       /* 0x62 b */
+    0x60, 0x80, 0x80, 0x60, 0x00,       /* 0x63 c */
+    0x20, 0x60, 0xA0, 0xA0, 0x60,       /* 0x64 d */
+    0x60, 0xA0, 0xC0, 0x60, 0x00,       /* 0x65 e */
+    0x20, 0x40, 0xE0, 0x40, 0x40,       /* 0x66 f */
+    0x60, 0xA0, 0xE0, 0x20, 0x40,       /* 0x67 g */
+    0x80, 0xC0, 0xA0, 0xA0, 0xA0,       /* 0x68 h */
+    0x80, 0x00, 0x80, 0x80, 0x80,       /* 0x69 i */
+    0x20, 0x00, 0x20, 0xA0, 0x40,       /* 0x6A j */
+    0x80, 0xA0, 0xC0, 0xC0, 0xA0,       /* 0x6B k */
+    0xC0, 0x40, 0x40, 0x40, 0xE0,       /* 0x6C l */
+    0xE0, 0xE0, 0xE0, 0xA0, 0x00,       /* 0x6D m */
+    0xC0, 0xA0, 0xA0, 0xA0, 0x00,       /* 0x6E n */
+    0x40, 0xA0, 0xA0, 0x40, 0x00,       /* 0x6F o */
+    0xC0, 0xA0, 0xA0, 0xC0, 0x80,       /* 0x70 p */
+    0x60, 0xA0, 0xA0, 0x60, 0x20,       /* 0x71 q */
+    0x40, 0x80, 0x80, 0x80, 0x00,       /* 0x72 r */
+    0x60, 0xC0, 0x60, 0xC0, 0x00,       /* 0x73 s */
+    0x40, 0xE0, 0x40, 0x40, 0x60,       /* 0x74 t */
+    0xA0, 0xA0, 0xA0, 0x60, 0x00,       /* 0x75 u */
+    0xA0, 0xA0, 0xE0, 0x40, 0x00,       /* 0x76 v */
+    0xA0, 0xE0, 0xE0, 0xE0, 0x00,       /* 0x77 w */
+    0xA0, 0x40, 0x40, 0xA0, 0x00,       /* 0x78 x */
+    0xA0, 0xA0, 0x60, 0x20, 0x40,       /* 0x79 y */
+    0xE0, 0x60, 0xC0, 0xE0, 0x00,       /* 0x7A z */
```

Comment:
I think you deleted a few too much now...
WLED expects the array to have all chars from 0x20 thru 0x7E 
>   if (chr < 32 || chr > 126) return; // only ASCII 32-126 supported

So your table needs to have everything up to 126 = 0x7E.
Please re-add the missing 4 chars. We want to avoid crashing due to array-out-of-bounds read.